Source/WebCore/Modules/mediastream/gstreamer/GStreamerStatsCollector.cpp

```diff
@@ -158,7 +158,7 @@ RTCStatsReport::InboundRtpStreamStats::InboundRtpStreamStats(const GstStructure*
 
 RTCStatsReport::OutboundRtpStreamStats::OutboundRtpStreamStats(const GstStructure* structure)
     : SentRtpStreamStats(Type::OutboundRtp, structure)
-    , remoteId(gstStructureGetString(structure, "remote-id"_s).toString())
+    , remoteId(String(gstStructureGetString(structure, "remote-id"_s).span()))
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?